browser/ui/views/split_view/split_view.cc

```diff
@@ -415,6 +426,10 @@ void SplitView::UpdateSecondaryContentsWebViewVisibility() {
   split_view_separator_->SetVisible(
       secondary_contents_container_->GetVisible());
 
+#if BUILDFLAG(ENABLE_SPEEDREADER)
+  UpdateSecondaryReaderModeToolbar(nullptr);
```

Comment:
do we need this call here?

Reply:
Here we need only update the visibility, so I added new method